hazelcast/src/main/java/com/hazelcast/map/impl/operation/PutFromLoadAllOperation.java

```diff
@@ -70,8 +71,12 @@ public void run() throws Exception {
             Object previousValue = recordStore.putFromLoad(key, value);
 
             callAfterPutInterceptors(value);
+            // Null-pointer here, in case of null key loaded by MapLoader
+            checkNotNull(key, "Key loaded by a MapLoader cannot be null.");
```

Comment:
Sure, I'm fully with you, but I didn't change any behaviour in this PR. The thing is that the code blows up in these places without the checks anyway. It's just a nasty NullPointerException on the `Record record = recordStore.getRecord(key);` So the line I added just prints a nicer message.

The only behaviour change that this PR is doing is not calling the interceptor if the value is null, but this is a clear bug fix, since earlier the interceptor was called even though the putFromLoad ignored this null value.